pages/index.vue

```diff
@@ -30,14 +33,19 @@ input {
     width: 80%;
     margin: 10px auto;
     border-radius: 20px;
+    padding: 5px 5px;
 }
 button {
     background: $primary500;
     border: 0;
     padding: 10px 50px;
-    margin-top: 20px;
+    margin-top: 30px;
     color: $text500;
     border-radius: 20px;
+    cursor: pointer;
+}
+button:hover {
+    background-color: $primary600;
```

Comment:
ogólnie składnia scss pozwala na ładniejsze style (z zagnieżdżaniem) ale to kiedyś zobaczysz jak się tak robi